SUPPORT.md

```diff
@@ -37,7 +37,7 @@ consider searching the mail archives and posting a question to the
 
 ### Open an Issue
 
-If you wish to report a bug, please open an [issue][github-issues] on GitHub
+If you wish to report a bug, please open an [issue](../../issues/new/choose) on GitHub
```

Comment:
I think you should make a link at the end of the document in the links section, like the other links in the document.

Reply:
Done, that does indeed make much more sense.